docs/modules/integrate/pages/cdc-connectors.adoc

```diff
@@ -39,28 +74,62 @@ For the setting up a streaming source of CDC data is just the matter of pointing
 Pipeline pipeline = Pipeline.create();
 pipeline.readFrom(
     MySqlCdcSources.mysql("customers")
-            .setDatabaseAddress("127.0.0.1")
-            .setDatabasePort(3306)
-            .setDatabaseUser("debezium")
-            .setDatabasePassword("dbz")
+            .setDatabaseAddress("127.0.0.1", 3306)
+            .setDatabaseCredentials("debezium", "dbz")
             .setClusterName("dbserver1")
-            .setDatabaseWhitelist("inventory")
-            .setTableWhitelist("inventory.customers")
+            .setDatabaseIncludeList("inventory")
+            .setTableIncludeList("inventory.customers")
             .build())
     .withNativeTimestamps(0)
     .writeTo(Sinks.logger());
 ```
 
+MySQL- and PostgreSQL-specific source builders contain methods for all major configuration setting and it guards if
+e.g. mutually exclusive options are not used. For generic source builder user must rely on Debezium's documentation
+to provide all necessary options.
+
 For an example of how to use CDC data see xref:pipelines:cdc.adoc[our tutorial].
 
+=== Common source builder functions
+[cols="m,a"]
+|===
+|Method name|Description
+
+|changeRecord()
+| Sets output type to ChangeRecord - a wrapper, providing most of the fields in
+strongly-typed manner.
+
+| json()
+| Sets output type to JSON - result stage will have `Map<String, String>` as it's type,
+where key is SourceRecord's key in json format and value is whole SourceRecord's value in json string.
+
+|customMapping(RecordMappingFunction<T>)
+| Sets output type to some arbitrary user type `T`. Mapping from `SourceRecord` to `T` will
+be done using provided function.
+
+|withDefaultEngine()
+|Sets preferred engine to default (non-async) one. This engine is single-threaded,
+but also older and more tested. For most stable results (e.g. no async offset restore) this engine should be preferred. For MySQL and PostgreSQL especially this engine makes the most sense, as MySQL and PostgreSQL Debezium connectors are single-threaded only.
+
+|withAsyncEngine()
+|Sets preferred engine to async one. This engine is multithreaded (if connector supports
+it), but you must be aware of it's async nature, e.g. offset restore after restart is done
+asynchronously as well, leading to sometimes confusing results.
+
+|setProperty(String, String)
+|Sets connector property to given value. There are multiple overloads, allowing to
+sets value to some `long`, `String` or `boolean`.
```

Comment:
I'm not sure about this one, "overloads" are not referred to elsewhere.

I have offered an alternative, but if I have not understood correctly, let me know!